### PR TITLE
Login with facebook field removed.

### PR DESCRIPTION
**Pull request by Trupti-AngDev**
# Description
Login with facebook field was additional field in registration form.

# Solution
Removed login with facebook field.

### CODE REVIEW

File: src/app/register/register.component.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.google-login{
+justify-content: center;
+}
```
Comment (devharp) [inline src/app/register/register.component.scss:3]: no need for this

File: src/app/register/register.component.html [hunk 2]
```diff
@@ -104,15 +104,12 @@ <h3>Register Account</h3>
                     </div>
                     <h6>Already have an account? <a href="login"> Login</a> Here</h6>
                     <div class="login-difarent-way">
-                      <div class="row g-4">
+                      <div class="row g-4 google-login">
```
Comment (devharp) [inline src/app/register/register.component.html:107]: use the bootstrap's class justify-center

**Review by devharp — COMMENTED** [review]
this above changes

File: src/app/register/register.component.html (hunk 2 above)
Comment (Trupti-AngDev) [reply src/app/register/register.component.html:107]: okay.

**Review by devharp — APPROVED** [review]

**Pull request merged**